week-5/Homework/mandatory/2-project/js/main.js

```diff
@@ -0,0 +1,97 @@
+//settings for blue button
+
+let blueBtn = document.querySelector("#blueBtn");
+blueBtn.addEventListener("click", blueBtnFunc);
+
+function blueBtnFunc() {
+  document.querySelector(".jumbotron").style.backgroundColor = `#588fbd`;
+  document.querySelector(
+    ".btn.btn-primary.btn-lrg"
+  ).style.backgroundColor = `#ffa500`;
+  document.querySelector(
+    ".btn.btn-secondary.btn-lrg"
+  ).style.backgroundColor = `black`;
+  document.querySelector(".btn.btn-secondary.btn-lrg").style.color = `white`;
+}
+
+//settings for orange button
+
+let orangeBtn = document.querySelector("#orangeBtn");
+orangeBtn.addEventListener("click", orangeBtnFunc);
+
+function orangeBtnFunc() {
+  document.querySelector(".jumbotron").style.backgroundColor = `#f0ad4e`;
+  document.querySelector(
+    ".btn.btn-primary.btn-lrg"
+  ).style.backgroundColor = `#5751fd`;
+  document.querySelector(
+    ".btn.btn-secondary.btn-lrg"
+  ).style.backgroundColor = `#31b0d5`;
+  document.querySelector(".btn.btn-secondary.btn-lrg").style.color = `white`;
+}
+
+//settings for green button
+
+let greengeBtn = document.querySelector("#greenBtn");
+greenBtn.addEventListener("click", greenBtnFunc);
+
+function greenBtnFunc() {
+  document.querySelector(".jumbotron").style.backgroundColor = `#87ca8a`;
+  document.querySelector(
+    ".btn.btn-primary.btn-lrg"
+  ).style.backgroundColor = `black`;
+  document.querySelector(
+    ".btn.btn-secondary.btn-lrg"
+  ).style.backgroundColor = `#8c9c08`;
+}
+
```

Comment:
Thanks for your feedback. Unfortunately, I can not make it work properly. I supposed that this function setColors should be anonymous, otherwise addEventListener calls it without button pressing. But if I use anonymous function, it will come to nothing the advantage of function. Could you give some advice how can I avoid this issue please? Thanks for you assistance. There is a code for a colour button. `let blueBtn = document.querySelector("#blueBtn");
blueBtn.addEventListener(
  "click",
  setColor("#588fbd", "#ffa500", "black", "white")
);
`